floodviz/static/js/hydrograph.js

```diff
@@ -1,17 +1,19 @@
-document.addEventListener("DOMContentLoaded", function (event) {
-	"use strict";
+"use strict";
```

Comment:
Should use the function form of "use strict"; See https://stackoverflow.com/questions/4462478/jslint-is-suddenly-reporting-use-the-function-form-of-use-strict for why.